src/main/webapp/election-info.js

```diff
@@ -180,6 +180,15 @@ function initializeMap() {
         zoom: 13
       });
       console.log("Created map");
+      
+      const homeMarker = new google.maps.Marker({
+        position:results[0].geometry.location, 
+        map:map,
+        icon: "https://img.icons8.com/ios-filled/50/000000/home.png",
```

Comment:
Does this render correctly on your side? Going to the site just shows black haha:
https://screenshot.googleplex.com/462d4c4f-50bb-49fe-be44-93890a02a358

Reply:
are you looking at the local site or deploying? I looked at the deployed site and the map loaded for me, but I can switch to this branch and deploy for reference right now.

Reply:
oh I misunderstood your question: here's what the icon looks like on the loaded site 
![image](https://user-images.githubusercontent.com/38237443/90665424-dcc50c00-e200-11ea-974a-e8fb7cad9309.png)

Reply:
If you look up links for embedding icons on google maps, the background is always black; it's a little confusing here because the icon is also black :) For example, checkout this marker: https://img.icons8.com/fluent/48/000000/marker-storm.png